configapi/api_inventory.go

```diff
@@ -178,13 +186,18 @@ func updateGnbInNetworkSlices(gnb configmodels.Gnb) error {
 	filterByGnb := bson.M{
 		"site-info.gNodeBs.name": gnb.Name,
 	}
-	return updateInventoryInNetworkSlices(filterByGnb, func(networkSlice *configmodels.Slice) {
+	statusCode, err := updateInventoryInNetworkSlices(filterByGnb, func(networkSlice *configmodels.Slice) {
 		for i := range networkSlice.SiteInfo.GNodeBs {
 			if networkSlice.SiteInfo.GNodeBs[i].Name == gnb.Name {
 				networkSlice.SiteInfo.GNodeBs[i].Tac = *gnb.Tac
 			}
 		}
 	})
+	if err != nil {
+		logger.ConfigLog.Errorf("failed to update gNB in network slices: %v", err)
```

Comment:
Should this be `%v` or `%+v`?
```suggestion
		logger.ConfigLog.Errorf("failed to update gNB in network slices: %+v", err)
```
NOTE: This comments is applicable in multiple places of this PR